### PR TITLE
Prepare release 0.5.4

### PR DESCRIPTION
**Pull request by pgnickb**
## Description
Release 0.5.4 #426 

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [x] CHANGELOG entry for user-facing changes
- [x] Updated the relevant documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ We use the following categories for changes:
 
 ## [Unreleased]
 
+## [0.5.4] - 2022-07-18
+
+Due to critical bug with extension upgrade to 0.5.3, we're skipping version 0.5.3 and incorporating the fix into 0.5.4. If you have installed 0.5.3 fresh you should have no isssue upgrading to 0.5.4. Upgrade paths to 0.5.3 are thus being removed. Images holding Promscale Extension 0.5.3 were purged from sources we have countrol over and we strongly advice anyone who managed to create such images to purge them for good.
```
Comment (JamesGuthrie) [inline CHANGELOG.md:19]: Can you break this paragraph at 80 chars?

> If you have installed 0.5.3 fresh you should have no isssue upgrading to 0.5.4.

I'm not sure that this is true. As it is, there is no upgrade path from 0.5.3 to 0.5.4 (no `promscale--0.5.3--0.5.4.sql` file).

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pgnickb) [reply CHANGELOG.md]: Does it look better now?

Comment (JamesGuthrie) [reply CHANGELOG.md]: Can you break the lines at 80 chars?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ We use the following categories for changes:
 
 ## [Unreleased]
 
+## [0.5.4] - 2022-07-18
+
+Due to critical bug with extension upgrade to 0.5.3, we're skipping version 0.5.3 and incorporating the fix into 0.5.4. Images holding Promscale Extension 0.5.3 were purged from sources we have countrol over and we strongly advice anyone who managed to create such images to purge them for good.
```
Comment (JamesGuthrie) [inline CHANGELOG.md:19]:
```suggestion
Due to a critical bug in upgrading to 0.5.3, we're skipping version 0.5.3 and
incorporating the fix into 0.5.4. We have removed the build artifacts for
Promscale Extension 0.5.3. We strongly advise anyone who used the 0.5.3 tag in
our GitHub or DockerHub repositories to delete these artifacts.```

**Review by JamesGuthrie — APPROVED** [review]

**Pull request merged**